### PR TITLE
Feat/deliverable2

### PR DESCRIPTION
**Pull request by eugenio-garcia**
Deliverable 2

### CODE REVIEW

File: src/Content.js [hunk 1]
```diff
@@ -1,10 +1,54 @@
...
+function Content({showProducts, setShowProducts}) {
+  
 
-function Content() {
   return (
     <div className="App-content">
-      <Home />
+      <If conditional={showProducts}>
+        <Products></Products>
+      </If>
```
Comment (julianzdelat) [inline src/Content.js]: If you want to return or not one component based in a condition I suggest you to do something like this:

```suggestion
        {showProducts && <Products/>}
```

File: src/components/Products.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    text-decoration: none;
+
+    &:hover {
+      background-color: ${(props) => props.theme.fg};
+      color: ${(props) => props.theme.bg};
+    }
+  `;
+
+  const handleChangeCategory = (event) => {
+    let category = event.target.name.toLowerCase();
```
Comment (julianzdelat) [inline src/components/Products.js]: if category is not going to be modified i suggest you to make the variable a constant

**Review by julianzdelat — CHANGES_REQUESTED** [review]
I am seeing also that the button that move the user to the filters is at the bottom, try to move this to a better place, also for the filters modify some of the json data in order to be able to see the filters working for all the categories.

About the categories, those re being appended when you are on the filters view, try to add an empty space between each one when are selected.

**Comment by eugenio-garcia** [comment]
@julianzdelat thanks for the feedback, I think this all has been addressed.

File: src/Content.js [hunk 3]
```diff
@@ -1,10 +1,49 @@
...
+  font-size: 1em;
+  margin: 1em;
+  padding: 0.25em 1em;
+  border-radius: 3px;
+`;
+
+const If = ({ conditional, children }) => {
+  if (conditional) return children;
+  return null;
+};
```
Comment (julianzdelat) [inline src/Content.js]: Since you are not using this function you can delete it to clean your code

Comment (eugenio-garcia) [reply src/Content.js]: Thanks, gonna remove it for now and use it later if needed.

**Review by julianzdelat — APPROVED** [review]
Thanks for addressing the comments

**Pull request merged**